pyleecan/Methods/Geometry/Line/comp_normal.py

```diff
@@ -0,0 +1,39 @@
+from numpy import angle, exp
+
+
+def comp_normal(self, return_type=2):
+    """Compute the normal direction of the Line
+    Normal point is "on top" (begin=O and end on Ox)
+
+    Parameters
+    ----------
+    self : Line
+        a Line object
+    return_type: int
+        0: (Zbegin+Zend)/2 (even for arc) and the normal point as tuple
+        1: Z2 - Z1 (with Z1 and Z2 the points from return_type 0)
+        2: the angle of the vector of return_type 1 according to Ox [rad]
```

Comment:
The varialbe return type seems strange to me. Since there are essentically only 3 relevant quantities (Z1, Z2, ang), I would return them all. Then every on how needs type 1 can do Z2 - Z1 themselve. 
Futher one can also 'filter' the return, e.g. comp_normal[2] for the angle only.

Reply:
Hello,

I agree, return type was more an experimentation at the beginning. But now I think that we can just change it to return always the second option because we can't have option 0 or 1 for HoleUD.